content/source/docs/enterprise/private/capacity.html.md

```diff
@@ -0,0 +1,71 @@
+---
+layout: "enterprise2"
+page_title: "Private Terraform Enterprise Capacity and Performance"
+sidebar_current: "docs-enterprise2-private-installer-capacity"
+---
+
+# Private Terraform Enterprise Capacity and Performance (Installer)
+
+The maximum capacity and performance of Private Terraform Enterprise (PTFE) is dependent entirely on the resources
+provided by the Linux instance it is installed on. There are a few settings that allow the capacity that PTFE
+uses to be adjusted to suit the instance.
+
+## Memory + Concurrency
+
+The amount of memory to allocate to a Terraform run and the number of concurrent runs are the primary elements in
+understanding capacity above the base services.
+
+By default, PTFE allocates 256 MB of memory to each Terraform run, with a default concurrency of 10 parallel runs.
+Therefore, by default PTFE requires 2.6 GB of memory reserved for runs.
+
+After factoring in the memory needed to run the base services that make up the application, the default memory footprint of PTFE is approximately 4 GB.
+
+### Settings
+
+The settings for per-run memory and concurrency are available in the dashboard on port 8800, on the Settings page, under the Capacity section. They can also be set via
+the [application settings JSON file when using the automated install procedure](https://www.terraform.io/docs/enterprise/private/automating-the-installer.html#available-settings).
+
+## Increasing Capacity
+
+To increase the number of concurrent runs, adjust the **Capacity** setting. Note that this setting is not limited by
+system checks; it depends on the operator to provide enough memory to the system to accommodate the requested
+concurrent capacity. For instance, if **Capacity** is set to `100`, the instance would require, at a minumum,
+26 GB of memory reserved for Terraform runs.
```

Comment:
If the capacity is set to 100 but there is never more than 10 concurrent runs, what is the behaviour? This is somewhat linked to the above question, how does PTFE handle over-commitment that's under-used.

Reply:
If the limit is set insanely high but only goes up to 10, then everything works fine. It's just that the system is basically at time bomb if it can't actually handle the high limit.